library/src/app/components/bank-account-management/bank-account-connect/bank-account-connect.component.ts

```diff
@@ -215,37 +216,24 @@ export class BankAccountConnectComponent implements OnInit {
   }
 
   plaidOnSuccess(public_token: string, metadata?: any): void {
-    function isValidAsset(asset: string | null | undefined) {
-      return typeof asset == 'string';
-    }
-    function isOnlyAccount(accounts: any[]) {
-      return accounts.length == 1;
-    }
-
-    const asset =
-      this.config.environment == 'demo'
-        ? 'USD'
-        : metadata.accounts[0].iso_currency_code;
-    const account = metadata.accounts[0];
-
-    console.log(this.config.environment);
-    console.log(
-      this.config.environment == 'demo'
-        ? 'USD'
-        : metadata.accounts[0].iso_currency_code
-    );
-    console.log(asset);
-    console.log(metadata.accounts);
-    console.log(isValidAsset(asset));
-    console.log(isOnlyAccount(metadata.accounts));
+    if (metadata.accounts.length == 1) {
+      const account = metadata.accounts[0];
 
-    if (isOnlyAccount(metadata.accounts) && isValidAsset(asset)) {
       this.configService
         .getBank$()
         .pipe(
           take(1),
           switchMap((bank) => {
-            if (bank.supported_fiat_account_assets!.includes(asset)) {
+            // Default asset to 'USD' for non-production banks
+            const asset =
+              bank.type == BankBankModel.TypeEnum.Sandbox
+                ? 'USD'
```

Comment:
Shouldn't this be the first supported asset that the bank supports? We could have 'CAD' banks.

Reply:
Sure. I'll make the change.